src/deluge/dsp/compressor/rms_feedback.cpp

```diff
@@ -28,9 +28,9 @@ RMSFeedbackCompressor::RMSFeedbackCompressor() {
 // 16 is ln(1<<24) - 1, i.e. where we start clipping
 // since this applies to output
 void RMSFeedbackCompressor::updateER(float numSamples, q31_t finalVolume) {
-
-	// int32_t volumePostFX = getParamNeutralValue(Param::Global::VOLUME_POST_FX);
-	float songVolumedB = logf(finalVolume);
+	// 33551360
+	//  int32_t volumePostFX = getParamNeutralValue(Param::Global::VOLUME_POST_FX);
```

Comment:
```suggestion
	//  int32_t volumePostFX = getParamNeutralValue(Param::Global::VOLUME_POST_FX);
	// We offset the final volume by a minuscule amount to avoid a finalVolume of zero resulting in NaNs propagating.
```